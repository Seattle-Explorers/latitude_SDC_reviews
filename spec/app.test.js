const supertest = require('supertest');

const app = require('../server/app.js');

const regeneratorRuntime = require('regenerator-runtime'); // babel needs this line here to play nice

const request = supertest(app);

beforeEach(async () => {
  jest.setTimeout(90000);
})

describe('Server path test', () => {

  test('Server should response to GET request', async (done) => {
    const response = await request.get('/api/:id');

    expect(response.status).toBe(200);
    done();
  });
});
