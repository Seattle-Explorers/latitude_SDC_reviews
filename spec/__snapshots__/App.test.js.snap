// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
.c1 {
  font-size: 1.5em;
  font-weight: bolder;
}

.c2 {
  display: grid;
  grid-template-columns: repeat(2,auto);
}

.c0 {
  font-family: sans-serif display:flex;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
     (
     reviews)
  </div>
  <div
    className="c2"
  />
</div>
`;
